cmd/admin-handlers.go

```diff
@@ -70,15 +70,31 @@ func (adminAPI adminAPIHandlers) ServiceStatusHandler(w http.ResponseWriter, r *
 		return
 	}
 
+	objLayer := newObjectLayerFn()
+	if objLayer == nil {
+		writeErrorResponse(w, ErrServerNotInitialized, r.URL)
+		return
+	}
+
 	// Fetch storage backend information
-	storageInfo := newObjectLayerFn().StorageInfo()
+	storageInfo := objLayer.StorageInfo()
```

Comment:
`status` is `mc` output - each command in `mc` reports 'status' - shouldn't be concerned with the API you are going to implement. It is just a full picture here, see `mc ls --json` for more details or `mc mirror --json`. 

I leave it upto to you send a string or some typed enum based on `iota`, it can be easily converted by `mc`.